source/spp/mnm/spp_117_nl_abiotiek_waterlopen/03_benodigdheden.Rmd

```diff
@@ -0,0 +1,33 @@
+# Extra benodigdheden t.o.v. de subprotocols
+
+Tabel \@ref(tab:Tabel1) geeft een overzicht van de benodigde apparatuur en materiaal voor het uitvoeren van een abiotische staalname.
+
++---------------------------------------------------------------------+-------------------------------------------------------------------+
+| Algemene benodigdheden                                              | Specifieke benodigdheden deelprotocollen                          |
++=====================================================================+===================================================================+
+| -   Staalnameformulier (analoog of digitaal)                        | -   Stroomsnelheidsmeter                                          |
++---------------------------------------------------------------------+-------------------------------------------------------------------+
+| -   Klembord, potlood en slijper                                    | -   Horizontale waterhapper, voorzien van een voldoende lang touw |
++---------------------------------------------------------------------+-------------------------------------------------------------------+
+| -   Laarzen, lieslaarzen, waadpak (site-afhankelijk)                | -   Telescopische steel met maatbeker                             |
++---------------------------------------------------------------------+-------------------------------------------------------------------+
+| -   Latex handschoenen (poederloos)                                 |                                                                   |
++---------------------------------------------------------------------+-------------------------------------------------------------------+
+| -   Plooimeter                                                      |                                                                   |
++---------------------------------------------------------------------+-------------------------------------------------------------------+
+| -   Fototoestel                                                     |                                                                   |
++---------------------------------------------------------------------+-------------------------------------------------------------------+
+| -   Documentatie van het staalnamepunt om dit te kunnen lokaliseren |                                                                   |
++---------------------------------------------------------------------+-------------------------------------------------------------------+
+
+: (#tab:Tabel1) Checklist benodigd veldmateriaal.
+
+## Materiaal
+
+Zie materiaal van de deelprotocollen ([sfp-113-nl 2023.04](../2023.04/index.html), [sfp-114-nl 2024.03](../2024.03/index.html), [sfp-115-nl 2024.04](../2024.04/index.html)), aangevuld met:
+
+-   Horizontale waterhapper (bijvoorbeeld Friedinger, Van Dorn, …) voorzien van nylonkoord of touw.
```

Comment:
toevoeging: "Afhankelijk van de veldomstandigheden is volgend materiaal aangewezen om waterstalen te nemen in stromend water: " en verwijzing naar paragraaf 5.5 Bemonstering waterkolom en het vullen van de recipiënten

Reply:
Toegevoegd!